README.md

```diff
@@ -30,11 +30,67 @@
 * The API itself is really simple. You're task is to peek at the endpoints found in the `server.js` file and write docs for each one, then write the tests for the end points.
 
 ## DOCUMENTATION GOES HERE
+# Server Testing - Games API Documentation
+This API was created in order to help understand Server testing using ```Chai```, ```Chai-HTTP``` and ```Sinon```.
 
+The API contains routes that use the following HTTP verbs: POST, GET, PUT and DELETE.
+
+It will allow users to view a list of games with their genre dateReleased and title. Users will also be able to create new game entries to the database, edit a given game and delete a given game.
+
+| Endpoint            | Type          | Data  |
+| ------------------- |:-------------:| -----:|
+| /api/game/create    | POST          | json  |
+
+### Example:
+```
```

Comment:
Something I want to put on your radar - Markdown supports syntax highlighting. To make use of this, after the opening triple backticks, if you put `js` or `bash` or `json` `ruby` `py` `c` et cetera, the markdown script will get rendered with the appropriate syntax highlighting/coloring. See lines 117-119 in this document:

```js
expect(res.data[0].foo).to.equal(bar.foo);
```

For more info on "fenced code blocks" and "syntax highlighting":
https://help.github.com/articles/creating-and-highlighting-code-blocks/

For more info on Markdown styling: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet